tests/conftest.py

```diff
@@ -75,7 +78,83 @@ def tmp(infile_path, outpath):
         return os.path.join(outpath, infile_name)
     return tmp
 
-
+@dataclass
+class MaterialData:
+    correlation_mat_names: List[str]
+    material_names: List[str]
+    temperature_range: Dict[str, np.array]
+    correlation_dict: Dict[str, Any]
+    correlation_dict_2: Dict[str, Any]
+    correlation_dict_wrong: Dict[str, Any]
+    expected_from_corr: Dict[str, List[float]]
+    coeff: Dict[str, List[float]]
+    bad_coeff: Dict[str, List[float]]
+    lead_interp: Dict[str, List[float]]
+    lbe_interp: Dict[str, List[float]]
+    bismuth_interp: Dict[str, List[float]]
+    sodium_interp: Dict[str, List[float]]
+    nak_interp: Dict[str, List[float]]
+    potassium_interp: Dict[str, List[float]]
+    ss304_interp: Dict[str, List[float]]
+    ss316_interp: Dict[str, List[float]]
+    lead_corr: Dict[str, List[float]]
+    lbe_corr: Dict[str, List[float]]
+    bismuth_corr: Dict[str, List[float]]
+    sodium_corr: Dict[str, List[float]]
+    nak_corr: Dict[str, List[float]]
+    properties_list: List[str] 
+    mat_from_tables: Dict[str, float]
+    out_range: Dict[str, List[float]]
+
+def pytest_configure(config):
+    """
+    Setup test data for the DASSH test suite
+    """
+    ###########################################
+    # Setup test data for material class tests
+    ###########################################  
+    with open("tests/test_data/material_class_data.json", "r") as json_file:
+        file_data = json.load(json_file)
+        
+    lead_bounds = lbh15.lead_properties.k().range
+    lbe_bounds = lbh15.lbe_properties.k().range
+    bismuth_bounds = lbh15.bismuth_properties.k().range
+        
+    out_range = {
```

Comment:
To avoid any hard-coded values, I'd suggest to substitute the last term of each value of the dictionary with the corresponding `lbh15` variable, that is: `lbh15.lead_properties.T_b0`, `lbh15.lbe_properties.T_b0` and `lbh15.bismuth_properties.T_b0`.